disk/datadog_checks/disk/disk.py

```diff
@@ -20,6 +20,13 @@
 IGNORE_CASE = re.I if platform.system() == 'Windows' else 0
 
 
+def _base_device_name(device):
```

Comment:
Let's invert the conditional to choose a function def

Reply:
Sorry  I don't understand what you mean.

Reply:
The OS will never change, so choose logic once at load time

Reply:
In fact, re-use the above `if platform.system() == 'Windows'`

Reply:
Oh I see, thanks.